enhancements/network/on-prem-configurable-lb-dns-stack.md

```diff
@@ -0,0 +1,316 @@
+---
```

Comment:
Should this new file go into the `network` directory? Or `baremetal`? Or somewhere else?

Reply:
Since it's relevant for all on-prem platforms I thought  it should go into network (like on-prem-service-load-balancers.md ) directory, 
let me know if you think it should land somewhere else.

Reply:
`network` makes sense to me.